doc/examples/sample-ml-app/README.txt

```diff
@@ -0,0 +1,38 @@
+This directory contains a Visual Studio solution showing the use of mlpack in a
+Visual Studio C++ project.
+
+However, you will need to set up your environment correctly first---or modify
+the project properties accordingly---in order to build and run the example.
+
+In short, you must download mlpack and four libraries, and install the sources
+and library files into C:\mlpack\.
+
+ * OpenBLAS: https://github.com/xianyi/OpenBLAS/releases/download/v0.3.21/OpenBLAS-0.3.21-x64.zip
+   Download the .zip, and extract it into C:\mlpack\openblas-0.3.21\
+
+ * Armadillo: https://mlpack.org/files/armadillo-11.4.1.tar.gz
+   Download the .tar.gz, and extract it into C:\mlpack\armadillo-11.4.1\; note
+   that you may need to use a program such as 7Zip ( ) to unpack this archive.
```

Comment:
`( )` is that a link you wanted to add?

Reply:
Thank you!  This is what code review is for I guess. :)  Link fixed in 5d1190a.